src/main/java/org/jabref/cli/Launcher.java

```diff
@@ -45,8 +45,10 @@
  */
 public class Launcher {
     private static Logger LOGGER;
+    private static String[] ARGUMENTS;
```

Comment:
Is this field really necessary?

Reply:
Just to make sure the original args are no modified